### ISSUE TITLE
sounds corrupt when coping multiple nodes from gms to v83

### ISSUE THREAD

**Issue by resinate** [lastbattle/Harepacker-resurrected#58]
all nodes are bugged sounds when copying more than 1 into v83 sound wz

**Comment by lastbattle** [comment]
Odd. It seems to be working just fine here when I coped 2 nodes and saving it.

![image](https://user-images.githubusercontent.com/4586194/101278016-5e15ce00-37f3-11eb-85a9-a5e58fa5a1a9.png)

**Comment by resinate** [comment]
yes copying 1 or 2 works fine, but when u copy like all monster sounds at once they break after like 250 nodes
